submissions/alstep07/task-js-dom/app.js

```diff
@@ -0,0 +1,333 @@
+        },
+        {
+         'title': 'Tumbbad',
+         'year': '2018',
+         'rating': '8.3',
+         'director': 'Rahi Anil Barve',
+         'description': 'A mythological story about a goddess who created the entire universe. The plot revolves around the consequences when humans build a temple for her first-born.'
+        },
+        {
+         'title': 'The Blue Elephant',
+         'year': '2014',
+         'rating': '8.1',
+         'director': 'Marwan Hamed',
+         'description': 'The story of Dr. Yehia, a psychotherapist at Al Abbasia hospital. He works in the department treating the criminally insane, only to find his best friend to be one of the patients. Trying to help his friend he finds himself going down the rabbit hole.'
+        }
+    ]
+};
+
+
+const main = document.querySelector('.main__container');
+const navBtn = document.querySelectorAll('.navbar__item');
+const genres = ['action', 'drama', 'comedy', 'western', 'romance', 'horror'];
+
+render('all');
+
+navBtn.forEach(function (btn,i){
+    btn.addEventListener('click', function(){
+        render(genres[i]);
+    })
+});
+
+function render(genre){
+
+    if (main.hasChildNodes()){
+        main.firstChild.remove();
+    }
+
+    let filmCards = document.createElement('div');
+    filmCards.className="main__genre";
+    
+    movies[genre].forEach( function(film,index){
+
+        let filmData = document.createElement('div');
+        let filmDescr = document.createElement('p');
+        let card = document.createElement('div');
+        let cover = document.createElement('img');
+        let title = document.createElement('h2');
+        let year = document.createElement('p');
+        let rating = document.createElement('p')
+        let descr = document.createElement('p');
+        let director = document.createElement('p');
+        
+        cover.setAttribute('src',`./images/${genre+index}.jpg`);
+        cover.setAttribute('alt', 'cover');
+
+        filmDescr.className = 'film__descr';
+        cover.className ='film__cover film__cover-active';
+        title.className ='film__title';
+        year.className ='film__year';
+        rating.className ='film__rating';
+        director.className ='film__director';  
+        filmData.className='film__data film__data-active';
+        
+        title.textContent=`${film['title']}`;
+        year.textContent=`year: ${film['year']}`;
+        rating.textContent=`imdb: ${film['rating']}`;
+        filmDescr.textContent=`${film['description']}`;
+        director.textContent = `${film['director']}`;
+        
+        filmData.appendChild(title);
+        filmData.appendChild(year);
+        filmData.appendChild(rating);
+        filmData.appendChild(director);
+        card.appendChild(cover);
+        card.appendChild(filmData);
+        card.appendChild(filmDescr);    
```

Comment:
With [append](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append) you can do it in one operation.
`filmData.append(el1, el2, ...)`